aemedge/blocks/popular-blogs/popular-blogs.css

```diff
@@ -1,12 +1,181 @@
-.section.popular-blogs-container
-{
-    background: #f2f2f2;
+.section.popular-blogs-container {
     margin: 0 auto;
     width: 100%;
     padding: 37px 0;
+    background: #f2f2f2;
+    margin-bottom: 50px;
+}
+
+.popular-blogs .slides-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.popular-blogs .slides-wrapper {
+    position: relative;
+}
```

Comment:
Add to this to make only the .slides-wrapper scroll and not the header text:
overflow-x: auto;
width: 100%;

Reply:
added these changes. seems now only slide wrapper is scrolling and not the entire section.